### DISCUSSION TITLE
Some inquiries about OrientDB

### DISCUSSION THREAD

**Discussion by wusf233** [orientechnologies/orientdb#9992] [General]
I would like to ask some questions about OrientDB, does OrientDB support multi-table operations now?

For example, the following SQL statement:

“UPDATE consumption_2,weibo SET consumption_2.product_type='update_type', weibo.gender = 'f' WHERE consumption_2.id=weibo.id AND weibo.id=5”

**Comment by tglman** [comment]
Hi,

No, basic target cannot have more than one class ( what you refer as table), the only statement that support multiple targets is the [pattern matching](https://orientechnologies.github.io/docs/3.2.x/sql/SQL-Match.html)  

Bye
